packages/api-clients/api-codegen-preset/src/tests/api-project.test.ts

```diff
@@ -28,7 +29,17 @@ describe('shopifyApiProject', () => {
 
         // THEN
         expect(projectConfig).toEqual({
-          schema: `https://shopify.dev/${type}-graphql-direct-proxy/2023-10`,
+          schema:
```

Comment:
There's a little bit of repetition here - I wonder if we could pull this into a private method?

Reply:
yup, added a helper